src/VisualStudio/TestSetup/BindingPath.pkgdef

```diff
@@ -0,0 +1,2 @@
+﻿[$RootKey$\BindingPaths\{d0122878-51f1-4b36-95ec-dec2079a2a84}]
+"$PackageFolder$"=""
```

Comment:
Should we be more strict and have codebase attributes for the specific assemblies we're using?